webapp/security/views.py

```diff
@@ -536,3 +513,57 @@ def delete_cve(cve_id):
         return flask.jsonify({"message": error.orig.args[0]}), 400
 
     return flask.jsonify({"message": "CVE deleted succesfully"}), 200
+
+
+@authorization_required
+def bulk_upsert_cve():
+    """
+    Receives a PUT request from load_cve.py
+    Parses the object and bulk inserts or updates
+    @returns 3 lists of CVEs, created CVEs, updated CVEs and failed CVEs
+    """
+
+    cves_data = flask.request.json
+    create = []
+    update = []
+
+    if not type(cves_data) == list:
+        return (
+            flask.jsonify(
+                {"message": "Please, submit a list (array) of CVEs"}
+            ),
+            400,
+        )
+
+    for data in cves_data:
+        cve = db_session.query(CVE).get(data["id"])
+        if cve:
+            update.append(data)
+        else:
+            create.append(data)
+
+    updated = update_cve(update)
+    created = create_cve(create)
+
+    try:
+        db_session.add_all(updated)
+        db_session.add_all(created)
+        db_session.commit()
+
+    except DataError as error:
+        return (
+            flask.jsonify(
+                {
+                    "message": "Failed bulk upserting session",
+                    "error": error.orig.args[0],
+                }
+            ),
+            200,
```

Comment:
This should not be `200`. This should be either `400 Bad Request` or `409 Conflict`. Up to you ([see the list here](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_errors)).